backend/packages/Upgrade/src/api/services/AnalyticsService.ts

```diff
@@ -81,12 +80,12 @@ export class AnalyticsService {
       groupExclusion,
     ] = promiseArray[1];
 
-    this.log.info('individualEnrollmentByCondition', individualEnrollmentByCondition);
-    this.log.info('individualEnrollmentConditionAndPartition', individualEnrollmentConditionAndPartition);
-    this.log.info('groupEnrollmentByCondition', groupEnrollmentByCondition);
-    this.log.info('groupEnrollmentConditionAndPartition', groupEnrollmentConditionAndPartition);
-    this.log.info('individualExclusion', individualExclusion);
-    this.log.info('groupExclusion', groupExclusion);
+    logger.info({ message : `individualEnrollmentByCondition ${individualEnrollmentByCondition}` });
```

Comment:
This will get logged as [Object Object]. Instead log this as another property of the object `data`
```
{
     message: "individualEnrollmentByCondition",
     data: individualEnrollmentByCondition
}
```

Reply:
okay